VimeoNetworking/Sources/Request+Category.swift

```diff
@@ -43,7 +43,7 @@ public extension Request
      
      - returns: a new `Request`
      */
-    public static func getCategoryRequest(categoryURI categoryURI: String) -> Request
+    public static func getCategoryRequest(forCategoryURI categoryURI: String) -> Request
```

Comment:
should the label just be for instead of forCategoryURI?  This seems to be the naming convention we're using everywhere else.